gui/include/settings.h

```diff
@@ -24,11 +24,22 @@
 
 #include <QSettings>
 
-class Settings : public QObject
-{
+typedef enum SettingsExtendedAnalogStickEnum {
+	// must not overlap with ChiakiControllerButton and ChiakiControllerAnalogButton
+	CHIAKI_CONTROLLER_ANALOG_STICK_LEFT_X_UP = (1 << 18),
+	CHIAKI_CONTROLLER_ANALOG_STICK_LEFT_X_DOWN = (1 << 19),
+	CHIAKI_CONTROLLER_ANALOG_STICK_LEFT_Y_UP = (1 << 20),
+	CHIAKI_CONTROLLER_ANALOG_STICK_LEFT_Y_DOWN = (1 << 21),
+	CHIAKI_CONTROLLER_ANALOG_STICK_RIGHT_X_UP = (1 << 22),
+	CHIAKI_CONTROLLER_ANALOG_STICK_RIGHT_X_DOWN = (1 << 23),
+	CHIAKI_CONTROLLER_ANALOG_STICK_RIGHT_Y_UP = (1 << 24),
+	CHIAKI_CONTROLLER_ANALOG_STICK_RIGHT_Y_DOWN = (1 << 25),
+} SettingsExtendedAnalogStickEnum;
```

Comment:
I think it would be better to differentiate these a bit more from the original definitions. I would do it like this (no typedef needed in C++):
```c++
enum class ControllerButtonExt {
	ANALOG_STICK_LEFT_X_UP = (1 << 18),
	...
};
```
Then later use it like `static_cast<int>(ControllerButtonExt::ANALOG_STICK_LEFT_X_UP)`